infrastructure/containers/postgres/automatic-backup/backup_or_restore.py

```diff
@@ -56,16 +60,50 @@ def run_shell_command(command, env_vars):
 ###################
 
 
-def make_backup_fpath():
-    return BACKUP_DIR / datetime.now().strftime(FILENAME_DATETIME_FORMAT)
+def make_object_name():
+    return datetime.now().strftime(FILENAME_DATETIME_FORMAT)
 
 
-def backup_database(db_backup_gzip_fpath):
+def make_backup_fpath(object_name):
+    return BACKUP_DIR / object_name
+
+
+def create_s3_client():
+    """Creates and returns an AWS S3 client for uploading backup to the cloud."""
+    sess = boto3.session.Session(
+        aws_access_key_id=BACKUP_DB__AWS_ACCESS_KEY_ID,
+        aws_secret_access_key=BACKUP_DB__AWS_SECRET_ACCESS_KEY,
+    )
+    return sess.client("s3")
+
+
+def create_s3_session():
```

Comment:
Should this function call the above function or vice-versa? Or do we even need both? Seems like there's some repeated, low-level logic here.

Reply:
I had the client func call the session func and reordered them for clarity